### DISCUSSION TITLE
No DNS published for info.meshcentral.com

### DISCUSSION THREAD

**Discussion by mitchplze** [Ylianst/MeshCentral#6227] [Issues] [answered]
**Describe the bug**
Not sure if I'm missing something super obvious here, but there is no DNS record published for info.meshcentral.com, which is where the [installation guide](https://meshcentral.com/docs/MeshCentral2InstallGuide.pdf) shows that the scripts live.

**To Reproduce**
Perform nslookup to info.meshcentral.com

**Expected behavior**
DNS record should be returned

**Screenshots**
![image](https://github.com/Ylianst/MeshCentral/assets/25162574/bc9ad1de-ba2d-4c31-81d1-a182e3266f9d)
![image](https://github.com/Ylianst/MeshCentral/assets/25162574/1b05db00-0b84-47f3-b2e1-e3b482ca2d16)
![image](https://github.com/Ylianst/MeshCentral/assets/25162574/a837ca5f-8be4-45eb-9d07-79455b9cff94)

**Comment by si458** [comment]
this is not a bug so ive moved it to a discussion.
the pdf docs are old and obsolute and should only be used as a reference, and chances are all the links are invalid now!
we have a new docs page here via github docs https://ylianst.github.io/MeshCentral/
the new web docs do need updating but its much more helpful now than the PDF !
p.s where did u find the link for the pdf? we need to remove the link

**Comment by mitchplze** [comment]
Okay thanks, I’ll check that out. The link is prominently in the GitHub README under “Installation Guide”

**Reply by si458** [reply, marked as answer]
🤦 yep should defo move that to the bottom of the readme saying 'ARCHIVED PDF' thanks!

**Comment by mitchplze** [comment]
Btw, the new docs _also_ have a link to the ancient user guide on the first page. 

https://ylianst.github.io/MeshCentral/install/

**Reply by si458** [reply]
thanks! will get it updated tomorrow, in the mean time, IF you fancied updating the docs, the code is all here hehe
https://github.com/Ylianst/MeshCentral/tree/master/docs/docs
and ill happily accept a PR request!

**Reply by mitchplze** [reply]
I gave it a shot! #6229
